src/components/Footer.css

```diff
@@ -0,0 +1,42 @@
+.footer-wrapper{
+	background-color: #515D66;
+	width: 100%;
+	/* height: 200px; */
+	color: #E0E1DD;
+	padding: 2rem;
+}
+.footer-container{
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+}
+
+.footer-links {
+	display: flex;
+	width: 50vw;
+	list-style: none;
+	justify-content: end;
+	/* margin-top: 2rem; */
+	/* margin-right: 2rem; */
+	font-size: 1.5rem;
+
+}
+
+.footer-link {
+	margin-left: 2.5rem;
+
+
+}
+
+.copywrite {
+	text-align: center;
+	margin-top: 4rem;
+
+}
+@media screen and (max-width: 800px) {
+	.footer-links {
+		flex-direction: column;
+		width: 40%;
+		margin-right: 2rem;
+	}
+}
```

Comment:
add
 @media screen and (max-width: 800px) {
+       .footer-links {
+               font-size: 1rem;
+       }
+}